target/linux/ipq40xx/base-files/etc/board.d/01_leds

```diff
@@ -56,6 +56,10 @@ engenius,ens620ext)
 glinet,gl-ap1300)
 	ucidef_set_led_netdev "wan" "WAN" "green:wan" "eth1"
 	;;
+linksys,whw03v2)
+    ucidef_set_led_default "power" "POWER" "pca963x:blue" "1"
+    ucidef_set_led_netdev "lan" "LAN" "pca963x:green" "eth0" "link"
```

Comment:
Triggering off eth0 will make it bling on any network traffic.

Reply:
I will remove it, I wanted to have a different color (RGB) during boot than once everything is up and running. 
The logic was: on power-on the light is blue and once it's running it become cyan-ish.

Reply:
Done